_data/master/navbars/reference.yml

```diff
@@ -107,6 +107,26 @@ toc:
   section:
   - title: RBAC for etcdv2
     path: /reference/advanced/etcd-rbac
+- title: "Calico's driver for OpenStack (networking-calico)"
+  section:
+  - title: networking-calico
+    path: /reference/networking-calico/
+  - title: Detailed Semantics
+    path: /reference/networking-calico/semantics
+  - title: Floating IPs
+    path: /reference/networking-calico/floating-ips
+  - title: Service IPs
+    path: /reference/networking-calico/service-ips
+  - title: DevStack
+    path: /reference/networking-calico/devstack
+  - title: Kuryr
```

Comment:
In this case I agree that 'Calico and Kuryr' can just as well be 'Kuryr' - because we don't want to put 'Calico and' in every title everywhere!

More generally though I don't think the sidebar names should be the same as titles, and there are many existing cases where they aren't.  It's often good for the sidebar content to be more abbreviated that the document title.